docs/painless/painless-guide/painless-datetime.asciidoc

```diff
@@ -335,6 +335,41 @@ if (zdt1.isAfter(zdt2)) {
 }
 ----
 
+==== Datetime Zone
+
+Both string datetimes and complex datetimes have a modifiable timezone with a
```

Comment:
The timezone isn't acutally modifiable, calling withX methods creates a new ZonedDateTime. For this reason, we also shouldn't use the term `change` below. Perhaps reword this to indicate a new ZDT is created?

Reply:
Good catch.  Fixed and added a note about immutability.